src/Atomic.Seo/Html/Validators/DescriptionValidator.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Atomic.Sео.Html.Validators
+{
+    internal static class DescriptionValidator
+    {
+        internal static bool HasDesctiption(this string? text)
+        {
+            return string.IsNullOrWhiteSpace(text);
+        }
```

Comment:
This method returns the opposite of what says. It also causes confusion when reading the code like if HasDescritption returns string.Empty in the Get method and if HasDescription in the GetDescription we get another description which is wrong. I would keep the HasDescription method name, make it return correct value and use ! where needed.